### DISCUSSION TITLE
Get injection directly

### DISCUSSION THREAD

**Discussion by AckslD** [bennypowers/nvim-regexplainer#31] [Ideas]
Hey, really cool plugin! I just watched your talk at neovim conf where you mention that you `get_node_at_cursor` and look up or down the heirarchy to find the regex node. Just a thought if it's easier, you could alternatively directly get the matches of the injection query for `regex`. See how I did that for example for `femaco` here:
https://github.com/AckslD/nvim-FeMaco.lua/blob/main/lua/femaco/edit.lua#L78

**Comment by bennypowers** [comment]
Thanks! Going to look into this, and hopefully make some improvements
